Application/EdFi.Ods.Api/EdFi.Ods.Api.csproj

```diff
@@ -66,6 +66,9 @@
   <ItemGroup>
     <EmbeddedResource Include="Services\Metadata\Resources\*.json" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Remove="Extensions\AuthorizationPolicyBuilderExtension.cs" />
```

Comment:
We shouldn't be removing C# files from compile, if we don't need them anymore we should delete them.